src/Symfony/Component/Notifier/Bridge/KazInfoTeh/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "symfony/kazinfoteh-notifier",
+  "type": "symfony-bridge",
+  "description": "Symfony KazInfoTeh Notifier Bridge",
+  "keywords": ["KazInfoTeh", "notifier", "symfony", "sms"],
+  "homepage": "https://symfony.com",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Egor Taranov",
+      "email": "dev@taranovegor.com",
+      "homepage": "https://taranovegor.com/contribution"
+    },
+    {
+      "name": "Symfony Community",
+      "homepage": "https://symfony.com/contributors"
+    }
+  ],
+  "require": {
+    "php": ">=8.0.2",
+    "ext-simplexml": "*",
+    "symfony/http-client": "^6.1",
```

Comment:
AFAIK you can use http client 5 too

Reply:
https://github.com/symfony/symfony/pull/44360#discussion_r762742880